src/bootstrap/bootstrap.py

```diff
@@ -620,6 +618,11 @@ def bootstrap():
     except:
         pass
 
+    if '\nverbose = 2' in rb.config_toml:
+        rb.verbose = 2
+    else if '\nverbose = 1' in rb.config_toml:
```

Comment:
Thanks! Fixed in 2aa6700b25a6979826020aef087d6611a1aab40a.

Wonder how this slipped through, though. I ran the build here without it stumbling over that SyntaxError...